packages/models/src/brand/createBrand.ts

```diff
@@ -0,0 +1,40 @@
+import { DescriptiveError, ErrorResponse } from "src/common/errorResponse";
+import { SuccessResponse } from "src/common/successResponse";
+
+export interface CreateBrandRequest{
+    id?: string,
+    info: Info,
+    meta: Meta
+}
+
+export interface Info{
+    name: string,
+    description?: string,
+    allowToSelectPageSize?: boolean,
+    published?: boolean,
+    displayOrder?: number,
+    pageSizeOptions?: number[] 
+}
+export interface Meta {
+    keywords?: string,
+    description?: string,
+    title?: string,
+    SEFN?: string
+}
+
+export interface CreateBrandSuccessResponse extends SuccessResponse {
+    code: number;
+    data: CreateBrandRequest
+}
+
+export interface CreateBrandErrorResponse extends ErrorResponse{
+    error: ErrorMessage.CANNOT_CREATE_BRAND | ErrorMessage.BRAND_ALREADY_EXISTS | ErrorMessage.NAME_REQUIRED | ErrorMessage.NAME_BE_VALID;
+}
+
+export const enum ErrorMessage{
+    CANNOT_CREATE_BRAND = 'CANNOT CREATE BRAND',
```

Comment:
can be the same Uppercase Error
like 
`CANNOT_CREATE_BRAND = 'CANNOT_CREATE_BRAND'`